scripts/create_gw_molecule_database.py

```diff
@@ -143,19 +156,20 @@ def write_geom_database(geom_info):
 
 @dataclass
 class GeomInfo():
-    output_geom_database_filepath: str = 'gw5000.txt'
-    mol_info_filepath: str = '/Users/vandanrevanur/personal/codes/chemistry/goDatabase/data/gw5000/df_5k.json'
+    number_of_molecules : int = 62000
```

Comment:
Shall we have a separate "input file" that can enter these parameters, or is it better to keep the "input file" combined with all the function code?

Reply:
For now I think we can leave in this input file itself, since they are all clubbed together in a dataclass, there is not much of a chance to miss anything when the user has to put in their data.

Reply:
Okay! I think all other threads are resolved, so I'll try to run this script to create the OE62 database for example! One complication that just came to my mind is the following: GW5000 is a subset of OE31which is a subset of OE62. So should the names be labeled with GW5000.v0 or with OE62.v0? I think OE62.v0, but then perhaps we could have a "tags" section or some other solution that could inform the computer that a particular molecule is not only part of OE62 but also part of GW5000 and OE31.  What do you think?